### PR TITLE
Adds scala/java tabs #30019

### PR DESCRIPTION
**Pull request by nvollmar**
References #30019

### CODE REVIEW

File: akka-docs/src/main/paradox/fault-tolerance.md [hunk 1]
```diff
@@ -280,13 +280,11 @@ And must be used for passivation:
 The following Scala snippet shows how to create a backoff supervisor which will start the given echo actor after it has stopped
```
Comment (raboof) [inline akka-docs/src/main/paradox/fault-tolerance.md:280]:
```suggestion
The following snippet shows how to create a backoff supervisor which will start the given echo actor after it has stopped
```

File: akka-docs/src/main/paradox/fault-tolerance.md [hunk 2]
```diff
@@ -301,13 +299,13 @@ crashes and the supervision strategy decides that it should restart.
 The following Scala snippet shows how to create a backoff supervisor which will start the given echo actor after it has crashed
```
Comment (raboof) [inline akka-docs/src/main/paradox/fault-tolerance.md:299]:
```suggestion
The following snippet shows how to create a backoff supervisor which will start the given echo actor after it has crashed
```

**Review by johanandren — APPROVED** [review]
Good, thanks!

**Review by raboof — COMMENTED** [review]
:+1: - did you test it in a browser? I always forget how many spaces are needed.

2 small suggestions

**Comment by nvollmar** [comment]
I formatted it the same way other snippets in the same file were

**Review by patriknw — APPROVED** [review]
LGTM, thanks

**Pull request merged**